ros2component/ros2component/api/__init__.py

```diff
@@ -248,3 +244,44 @@ def __init__(self, *, package_name_key=None):
     def __call__(self, prefix, parsed_args, **kwargs):
         package_name = getattr(parsed_args, self.package_name_key)
         return get_package_component_types(package_name=package_name)
+
+
+def add_component_arguments(parser):
+    """Add component specific arguments to the CLI parser."""
+    argument = parser.add_argument(
+        'package_name', help='Name of the package where the component is to be found'
+    )
+    argument.completer = package_with_components_name_completer
+    argument = parser.add_argument(
+        'plugin_name', help='Type name of the component to be loaded'
+    )
+    argument.completer = ComponentTypeNameCompleter(package_name_key='package_name')
+    parser.add_argument('-n', '--node-name', default=None, help='Component node name')
```

Comment:
Nitpick: `default=None` doesn't have to be specified, it is the default already.

Same below.

Reply:
Alright, too explicit maybe.